### DISCUSSION TITLE
What are your favorite `.vercel.app` sites?

### DISCUSSION THREAD

**Discussion by leerob** [vercel/community#3] [Show & Tell]
Could be anything you find interesting/helpful/cool 😄 

- https://swag.vercel.app
- https://bitcoin.vercel.app
- https://color.vercel.app
- https://carbon.vercel.app
- https://og-image.vercel.app
- https://signal.vercel.app
- https://my-room-in-3d.vercel.app/
- https://programming-language-benchmarks.vercel.app/

**Comment by delbaoliveira** [comment]
- https://swr.vercel.app/ 😉 
- https://nextra.vercel.app/ 😉 
- https://winxp.vercel.app/
- https://i2i-covid.vercel.app/
- https://signal.vercel.app/
- https://blaze.vercel.app/

**Reply by virtfav-design** [reply]
Hi! Can you help me how to create links like this to my website?

**Comment by analuzcervantes** [comment]
- https://kron.vercel.app/
- https://cakaren.vercel.app/
- https://meal.vercel.app/
- https://finesounds.vercel.app/
- https://www.acresceimoveis.com.br/
- https://winxp.vercel.app/

**Comment by leerob** [comment]
More!

- https://github-contributions.vercel.app
- https://bashplate.vercel.app
- https://pompom.vercel.app
- https://next-news.vercel.app

**Comment by leerob** [comment]
Even more!

- https://search-whats-missing-from-css.vercel.app
- https://lukyshawty.vercel.app
- https://blurhash-playground.vercel.app

**Comment by leerob** [comment]
https://kwote.app/

**Reply by rafreid** [reply]
https://gourdex.vercel.app

**Comment by Babbili** [comment]
- [https://flipcoins.vercel.app](https://flipcoins.vercel.app/)

**Comment by steven-tey** [comment]
Super biased, but...

https://dub.vercel.app/

**Comment by Andre601** [comment]
I like https://simpleicons.vercel.app/ because it inspired me to make my own version for GitHub's Octicons (https://colored-octicons.vercel.app/)

**Comment by LitoMore** [comment]
https://cdn.simpleicons.org

This one is made by Simple Icons official.

**Comment by PaulNext** [comment]
https://0x-next-labs.vercel.app
**Redirect to** https://nextme.one
The Social Economy Network Connecting Everything.
🐣 Submit to Gallery → [http://nextme.one/explore](https://t.co/FuR1Jcl41w)
🦄 Official Portal → [http://nextme.one/nextme.eth](https://t.co/bYKDcwc8QP)

![image](https://github.com/vercel/community/assets/100060836/2e49a7c1-9b72-455b-8d22-9f32e7df5e08)

**Comment by imran-baitham** [comment]
- https://fnlondon.vercel.app
- https://nozbe.vercel.app
- https://gbrent.vercel.app
- https://emojidb.vercel.app
- https://gitclone.vercel.app
- https://baitham.vercel.app
- https://linkty.vercel.app
- https://vist.vercel.app
- https://webskills.vercel.app

**Comment by imran-baitham** [comment]
- https://robot-sigma-nine.vercel.app
- https://tripperspk.vercel.app
- https://northaims.vercel.app
- https://book-cler.vercel.app
- https://react-query-instagram.vercel.app
- https://imran.vercel.app
- https://imranwebsite.vercel.app

## I love using @vercel and Vercel's free domains for my CV or practice projects. Here are a few of my projects that I'd like to share. Thanks to the @vercel and @nextjs teams for their amazing work!

**Comment by yuxxeun** [comment]
This site as, always:
https://yuxxeun.now.sh

**Comment by ronnelmusca** [comment]
#3154

**Comment by hmartiins** [comment]
www.graodegente.com.br
An e-commerce entirely under next js and vercel

**Comment by Pyr33x** [comment]
https://next-route-visualizer.vercel.app
https://app-router.vercel.app
https://swr.vercel.app
https://kwote.app

**Comment by mikedidomizio** [comment]
https://vercel.vercel.app

**Comment by lilac** [comment]
[Curateds](https://www.curateds.com/) is a community for curated lists, a platform to explore and curate awesome lists on various topics. It includes 500+ awesome lists and 100,000+ curated items. https://www.curateds.com/

It uses Vercel for deployment preview.

**Comment by misaelnieto** [comment]
https://www.noenieto.com/demos/lm317-calculator/

A React app hosted on vercel to calculate resistor values for the venerable LM317 voltage regulator.

**Comment by ghost** [comment]
https://snapcode.vercel.app

**Reply by misaelnieto** [reply]
Looks really neat!

**Comment by trikibilel** [comment]
https://expod.vercel.app
